README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img height="300px" src="./logo-text.svg" alt="logo"></p>
```

Comment:
Could you move all images used in the documentation to /art, as in the Open Source template?